includes/managers/controls.php

```diff
@@ -798,6 +801,15 @@ public function add_control_to_stack( Controls_Stack $element, $control_id, $con
 			'index' => null,
 		];
 
+		$control_type = 'controls';
+		if (
+			Performance::is_optimized_control_loading_feature_enabled()
+			&& Performance::should_optimize_controls()
```

Comment:
why `should_optimize_controls` doesn't return whether the feature is enabled?

```suggestion
			Performance::should_optimize_controls()
```

Reply:
It's not part of the experiment.